### DISCUSSION TITLE
/home/runner/work/_temp/7c87286d-9d0f-4068-9fad-0d86010125d6.sh: line 1: unexpected EOF while looking for matching `'' Error: Process completed with exit code 2.

### DISCUSSION THREAD

**Discussion by Kalmai221** [vercel/community#1294] [Help]
### Summary

I use Github to publish my code to Vercel. I keep on getting an error when I try publish. This only has happend on 19/01/2023 

### Additional information

```
Run vercel deploy --prebuilt --prod --token=secret-token p I'm
/home/runner/work/_temp/7c87286d-9d0f-4068-9fad-0d86010125d6.sh: line 1: unexpected EOF while looking for matching `''
Error: Process completed with exit code 2.
```

### Example

Try and publish something

**Comment by swarnava** [comment]
Hey, is this coming on Vercel Build logs or your CI pipeline? Doesn't look related to Vercel
